src/StochasticNoiseSimulator.cpp

```diff
@@ -170,33 +174,31 @@ std::map<std::string, double> StochasticNoiseSimulator::StochSimulate() {
 }
 
 void StochasticNoiseSimulator::runStochSimulationForId(unsigned int                                stochRun,
-                                                       int                                         n_qubits,
-                                                       dd::vEdge                                   rootEdgePerfectRun,
+                                                       int                                         nQubits,
                                                        std::vector<double>&                        recordedPropertiesStorage,
                                                        std::vector<std::tuple<long, std::string>>& recordedPropertiesList,
+                                                       std::map<std::string, int>&                 classicalMeasurementsMap,
                                                        unsigned long long                          localSeed) {
     std::mt19937_64                        generator(localSeed);
     std::uniform_real_distribution<dd::fp> dist(0.0, 1.0L);
 
-    const unsigned long numberOfRuns = stochastic_runs / max_instances + (stochRun < stochastic_runs % max_instances ? 1 : 0);
+    const unsigned long numberOfRuns = stochasticRuns / max_instances + (stochRun < stochasticRuns % max_instances ? 1 : 0);
     const int           approx_mod   = std::ceil(static_cast<double>(qc->getNops()) / (step_number + 1));
 
-    //        dd::NoiseOperationTable<dd::mEdge> noiseOperationTable(getNumberOfQubits());
-
     //printf("Running %d times and using the dd at %p, using the cn object at %p\n", numberOfRuns, (void *) &localDD, (void *) &localDD->cn);
     for (unsigned long current_run = 0; current_run < numberOfRuns; current_run++) {
         const auto t1 = std::chrono::steady_clock::now();
 
-        std::unique_ptr<dd::Package<>> localDD = std::make_unique<dd::Package<>>(getNumberOfQubits());
+        std::unique_ptr<StochasticNoiseSimulatorDDPackage> localDD = std::make_unique<StochasticNoiseSimulatorDDPackage>(getNumberOfQubits());
```

Comment:
avoid repeating the return type
```suggestion
        auto localDD = std::make_unique<StochasticNoiseSimulatorDDPackage>(getNumberOfQubits());
```

Reply:
done